applications/main/nfc/scenes/nfc_scene_mf_classic_dict_attack.c

```diff
@@ -5,6 +5,8 @@
 
 #define TAG "NfcMfClassicDictAttack"
 
+// TODO: Update progress bar with nested attacks
```

Comment:
TODOs will fail CI/CD lint stage, normally we don't leave them without appropriate issue in task tracker.